ts/features/itwallet/common/hooks/useItwDismissalDialog.tsx

```diff
@@ -2,41 +2,71 @@ import { Alert } from "react-native";
 import { useHardwareBackButton } from "../../../../hooks/useHardwareBackButton";
 import I18n from "../../../../i18n";
 import { useIONavigation } from "../../../../navigation/params/AppParamsList";
+import {
+  ItwDismissalContext,
+  trackItwDismissalAction,
+  trackItwDismissalContext
+} from "../../analytics";
 
 type ItwDismissalDialogProps = {
   handleDismiss?: () => void;
   customBodyMessage?: string;
+  dismissalContext?: ItwDismissalContext;
 };
 
+const confirmText = I18n.t("features.itWallet.generic.alert.confirm");
+const cancelText = I18n.t("features.itWallet.generic.alert.cancel");
```

Comment:
Do we have advantages doing this here? I prefer to have i18n strings inside components, for consistency with other code.